style.css

```diff
@@ -117,3 +117,439 @@ nav .nav-icon {
   padding: 0 0 0 0;
   margin: 0;
 }
+
+.recent-works h1 {
+  width: 327px;
+  height: 83px;
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 52px;
+  align-items: center;
+  color: #1c1a19;
+  margin: 120px 24px 0 24px;
+}
+
+.recent-works hr {
+  width: 327px;
+  height: 0;
+  border: 1px solid #212121;
+  margin: 24px 24px 0 24px;
+}
+
+.recent-works img {
+  width: 327px;
+  height: 237px;
+  margin: 80px 24px 0 24px;
+}
+
+.recent-works h2 {
+  width: 327px;
+  height: 36px;
+  font-family: 'Crete Round'sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  align-items: center;
+  color: #091e42;
+  margin: 24px 24px 0 24px;
+}
+
+.recent-works p {
+  width: 327px;
+  height: 147px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #091e42;
+  margin: 12px 24px 0 24px;
+}
+
+.recent-works ul {
+  list-style: none;
+  display: flex;
+  justify-content: flex-start;
+  margin: 12px 54px 0 24px;
+  padding: 0;
+}
+
+.recent-works li {
+  background: #fff;
+  border: 1px solid #8993a4;
+  margin-right: 12px;
+  padding: 0;
+}
+
+.recent-works .css {
+  align-items: center;
+  padding: 10px 12px;
+  width: 25px;
+  height: 20px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+}
+
+.recent-works .html {
+  width: 32px;
+  height: 20px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  padding: 10px 12px;
+  align-items: center;
+}
+
+.recent-works .boostrap {
+  width: 71px;
+  height: 20px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  padding: 10px 12px;
+  align-items: center;
+}
+
+.recent-works .ruby {
+  width: 37px;
+  height: 20px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  padding: 10px 12px;
+  align-items: center;
+}
+
+.recent-works button {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  align-items: center;
+  text-align: center;
+  padding: 12px;
+  gap: 10px;
+  width: 124px;
+  height: 48px;
+  background: #ff6b00;
+  color: #fff;
+  margin: 12px 0 0 24px;
```

Comment:
The button of the main project looks great. Kindly adjust it a bit to match the design having no black borders.

Your design:
![image](https://user-images.githubusercontent.com/20554583/199913736-faf33e9d-81f2-4fc9-a2d3-efbf4e94c91e.png)

Figma Design:
![image](https://user-images.githubusercontent.com/20554583/199913868-b1ecbbeb-59cf-4eed-9ed8-767498b02a9f.png)